tron/utils/rookout.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import platform
+from typing import Optional
+
+import rook
+
+import tron
+
+
+ROOKOUT_TOKEN: Optional[str] = None
+ROOKOUT_TOKEN_PATH = os.getenv("ROOKOUT_TOKEN_PATH", "/nail/etc/tron_rookout_key.txt")
+ROOKOUT_CONTROLLER_HOST = os.getenv("ROOKOUT_CONTROLLER_HOST", "169.254.255.254")
+ROOKOUT_CONTROLLER_PORT = int(os.getenv("ROOKOUT_CONTROLLER_PORT", 20798))
+ROOKOUT_ENABLE = os.getenv("ROOKOUT_ENABLE", False)
+
+log = logging.getLogger(__name__)
+
+
+def get_version_sha() -> Optional[str]:
+    version_sha = None
+    try:
+        with open(f"{os.path.dirname(os.path.abspath(__file__))}/../VERSION_SHA") as file:
+            version_sha = file.read().rstrip()
+    except OSError:
+        log.exception("Failed to read the file VERSION_SHA. Using master as default.")
+    return version_sha
+
+
+def get_hostname() -> str:
+    return platform.node()
+
+
+def prepare_rookout_token() -> None:
+    """Load rookout token into memory"""
+    global ROOKOUT_TOKEN
+    try:
+        with open(ROOKOUT_TOKEN_PATH, encoding="utf-8") as _rookout_token_file:
+            ROOKOUT_TOKEN = _rookout_token_file.read().strip()
+    except OSError:
+        log.exception("Failed to load rookout token")
```

Comment:
hmm, thoughts on having this function return the token (or None)? 

if we do that, then we don't need to use `global` anywhere (which is usually something to be avoided if possible)

Reply:
yup I was thinking to do that as well to avoid global. Good call  👍